Automattic/jetpack
# Term Checksums - fix item count discrepancy

Continued testing of the Checksum/Fix routine has revealed that the item_count for terms table are inconsistent. This is due to term_ids belonging to multiple term_taxonomy and having duplicate entries in the terms table. This patch ensures that distinct is used for the item_count query allowing us to have an accurate value for reporting of Accuracy.

#### Changes proposed in this Pull Request:

Add distinct clause for terms table item counts


#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Go to 'wp shell'
* $store = new Automattic\Jetpack\Sync\Replicastore();
* $store->checksum_histogram( 'terms', 1, null, null, null, true, '', false, true, false );

verify that the item_count returned matches the number of entries in the histogram.
